include/zephyr/bluetooth/conn.h

```diff
@@ -601,6 +641,31 @@ int bt_conn_le_set_tx_power_report_enable(struct bt_conn *conn,
 					  bool local_enable,
 					  bool remote_enable);
 
+/** @brief Set Path Loss Monitoring Parameters.
+ *
+ *  Change the configuration for path loss threshold change events for a given conn handle.
+ *
+ *  @param conn  Connection object.
+ *  @param param Path Loss Monitoring parameters
+ *
+ *  @return Zero on success or (negative) error code on failure.
+ */
+int bt_conn_le_set_path_loss_monitoring_parameters(struct bt_conn *conn,
+				const struct bt_conn_le_path_loss_reporting_parameters *param);
```

Comment:
Indentation looks weird here, but may always look weird due to the length of the struct name.

Consider shortening some of the names to e.g.
```suggestion
int bt_conn_le_set_path_loss_mon_param(struct bt_conn *conn,
				const struct bt_conn_le_path_loss_param *param);
```